enacts/flex_fcst/maproom.py

```diff
@@ -634,9 +660,13 @@ def fcst_tiles(tz, tx, ty, proba, variable, percentile, threshold, start_date, l
     # Reading
     
     if CONFIG["forecast_mu_file_pattern"] is None:
-        fcst_mu, fcst_var, obs = cpt.read_pycptv2dataset(DATA_PATH)
+        fcst_mu, fcst_var, obs = cpt.read_mpycptv2dataset(DATA_PATH)
         fcst_mu = fcst_mu.sel(S=start_date)
         fcst_var = fcst_var.sel(S=start_date)
+        if "L" in fcst_mu.dims:
+            fcst_mu = fcst_mu.sel(L=int(lead_time))
+            fcst_var = fcst_var.sel(L=int(lead_time))
```

Comment:
Instead of converting these here, mark them as int in line 657, as we've already done for tx, ty, tz,

Reply:
I think can't do that for now because read_cptdateset accepts lead_time as the lead time as a number, or as the target season as a string of characters, and so if the first case could be cast in a int right away, the second case can't. That was because the tsv case had written in the filename either the numerical lead or the target season (e.g. Oct-Dec)...

So I am afraid we'll have to stick with that until we revisit the tsv reading case.